src/libraries/System.Runtime.Loader/tests/ApplyUpdateTest.cs

```diff
@@ -39,6 +39,33 @@ void StaticMethodBodyUpdate()
             });
         }
 
+        [ConditionalFact(typeof(ApplyUpdateUtil), nameof (ApplyUpdateUtil.IsSupported))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/54617", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT))] 
```

Comment:
Is this wasm aot specific or should it be disabled on all mono aot?

Reply:
It's not going to work on any Mono AOT configuration - but ApplyUpdateUtil.IsSupported is already ruling out all the others.